test/examples/src/ts/UpdateDoc.ts

```diff
@@ -14,53 +14,55 @@
  * limitations under the License.
  */
 
-/* tslint:disable:no-console */
-
-import {CloudantV1} from "../../../../index";
+import { CloudantV1 } from '../../../../index';
 
 interface OrderDocument extends CloudantV1.Document {
-    address?: string;
-    joined?: string;
-    _id?: string;
-    _rev?: string;
+  address?: string;
+  joined?: string;
+  _id?: string;
+  _rev?: string;
 }
 
 // 1. Create a client with `CLOUDANT` default service name ================
 const client = CloudantV1.newInstance({});
 // 2. Update the document =====================================================
 // Set the options to get the document out of the database if it exists
-const exampleDbName = "orders";
+const exampleDbName = 'orders';
 
 // Try to get the document if it previously existed in the database
-const getDocParams: CloudantV1.GetDocumentParams =
-    {docId: "example", db: exampleDbName};
+const getDocParams: CloudantV1.GetDocumentParams = {
+  docId: 'example',
+  db: exampleDbName,
+};
 
-client.getDocument(getDocParams)
-    .then(docResult => {
-        // using OrderDocument on getDocument result:
-        const document: OrderDocument = docResult.result;
+client
+  .getDocument(getDocParams)
```

Comment:
Nit: I liked more the original formatting here.

Reply:
I don't think I did any formatting here manually. Do you want some config changes?

Reply:
It is just weird that here we see
```js
client
  .getDocument(getDocParams)
  .then((docResult) => {
  ...
  }
```
but some lines later (where the param list is longer than the previous) is:
```js
client.postDocument({ db: exampleDbName, document }).then((res) => {
...
}
```

It just does not seem to be consistent for me. Could you make it consistent somehow?

Btw the original formatting tried to look like this where it was preferred to keep the client and its method calling together but the async response handling separated by a new line:
```js
client.postDocument({ db: exampleDbName, document })
  .then((docResult) => {
  ...
  }
```
```js
client.getDocument(getDocParams)
  .then((docResult) => {
  ...
  }
```

Reply:
As per our conversation offline, I'll merge this as-is.
+CC @mojito317

Reply:
I understand you state that, there is no formatting change here, but still so many difference generated here: https://github.com/IBM/cloudant-node-sdk/pull/179/files
WDYT?

Reply:
I'm not seeing any formatting changes in https://github.com/IBM/cloudant-node-sdk/pull/179/files .
See 
![image](https://user-images.githubusercontent.com/9086834/109477045-dbaa4c80-7a77-11eb-9783-c27fef0e5f79.png)
(Whitespace changes are enabled for me: 
![image](https://user-images.githubusercontent.com/9086834/109477071-e664e180-7a77-11eb-8d6d-1ec292fd91f9.png) )

Can you please clarify, @vmatyusGitHub ?

Reply:
This was resolved already. The example reformatting from this PR had not been delivered via a README update PR - that merge happened in #182. Once that was done #179 was rebuilt and was no longer "polluted" with those undelivered changes.